Sources/SmithyHTTPAuth/SigV4Signer.swift

```diff
@@ -127,7 +128,7 @@ public class SigV4Signer: SmithyHTTPAuthAPI.Signer, @unchecked Sendable {
             signedBodyHeader: signedBodyHeader,
             signedBodyValue: signedBodyValue,
             flags: flags,
-            date: Date(),
+            date: Date().addingTimeInterval(clockSkew),
```

Comment:
Clock skew is applied to current date in the signing config to adjust signature expiration for the difference in client vs. server clock time.